provision/ansible/playbooks/install-sync-gateway-package.yml

```diff
@@ -36,7 +36,8 @@
 
   tasks:
   - include: tasks/flush-server-buckets.yml
-
+    when: skip_bucketflush == "false"
+    
```

Comment:
That looks right to me .. it only flushes server buckets if *skipping* bucketflush is false.  

I agree though it's a bit confusing.  

These variations would be less confusing:

```
   tasks:
    - include: tasks/flush-server-buckets.yml		    
      unless: skip_bucketflush == "true"
```

```
   tasks:
    - include: tasks/flush-server-buckets.yml		    
      when: do_bucketflush == "true"
```